examples/widgets/launch_url.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() {
+  runApp(new GestureDetector(
+    onTap: () {
+      Intent intent = new Intent()
+        ..action = 'android.intent.action.VIEW'
+        ..url = 'http://flutter.io/';
+      activity.startActivity(intent);
+    },
+    child: new Container(
+      decoration: new BoxDecoration(
+        backgroundColor: Colors.green[800]
```

Comment:
If you just hard code a `const Color()` you can make this depend on widgets instead of material, which seems reasonable given that this is in no way a material app. :-)